src/__tests__/Box.js

```diff
@@ -38,4 +38,10 @@ describe('Box', () => {
     expect(json).toMatchSnapshot()
     expect(json).toHaveStyleRule('background-color', theme.colors.green)
   })
+
+  test('boxShadowSize prop sets box shadow', () => {
+    const json = renderer.create(<Box boxShadowSize='small' />).toJSON()
+    expect(json).toMatchSnapshot()
+    expect(json).toHaveStyleRule('box-shadow', '0 0 4px 0 rgba(0,0,0,0.08)')
+  })
```

Comment:
Would you think the test cases can cover all the box shadow size alias from `small` to `xlarge`?

For box shadow aliases, would you think using short-hand terms such as `sm`, `md`,  `lg`, and `xl`? This will align to the naming convention in similar aliases in theme:
https://github.com/pricelinelabs/design-system/blob/master/src/theme.js#L26